scripts.py

```diff
@@ -600,11 +612,6 @@ def arg_parser(cls):
         )
         return parser
 
-    @classmethod
```

Comment:
This was overwriting the parent class's method with the same code, so I nixed it.